scst-scan/ivs-custom-samples.hbs.md

```diff
@@ -0,0 +1,54 @@
+# Configure your custom ImageVulnerabilityScan samples for Supply Chain Security Tools - Scan
+
+This topic provides you with sample ImageVulnerabilityScans for various scanners, and their associated secrets if required.
```

Comment:
Hi docs team, please let us know if improvements need to be made to this line. Thank you!